NOTICES

```diff
@@ -223,18 +223,11 @@ aeson 1.5.5.1 : BSD 3-clause "New" or "Revised" License
 aeson-compat 0.3.9 : BSD 3-clause "New" or "Revised" License
 aeson-extra 0.4.1.3 : BSD 3-clause "New" or "Revised" License
 aeson-pretty 0.8.8 : BSD 3-clause "New" or "Revised" License
-akka-actor 2.6.11 : Apache License 2.0
```

Comment:
what happened to akka :thinking:

Reply:
I suspect blackduck’s index is lagging behind and they don’t know about the new version 2.6.12 yes so they decided to just omit it completely …

Reply:
2.6.11 should be there 
![image](https://user-images.githubusercontent.com/25179017/106513671-40867d00-64a1-11eb-8eb9-b542b91420f7.png)

https://digitalasset.blackducksoftware.com/api/components/74f9a43f-8d82-40e8-9ae5-412d6ac0de1f/versions/c568a26e-4563-4d51-a498-22d72533b6bf

Reply:
i think we should rerun / ignore this update

Reply:
2.6.11 is there but we upgraded to 2.6.12. It looks like blackduck discards libraries whose version it doesn’t know which seems very questionable to me. fwiw, in today’s update they are still removing akka completely https://github.com/digital-asset/daml/pull/8710/files